bin/create-release

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env python3
+
+import os
+import re
+import requests
+import subprocess
+import shlex
+import sys
+
+# DOC: Create a new draft CiviForm image release. Usage: bin/create-release COMMIT_SHA RELEASE_NUMBER
+#
+# Requirements:
+#   Must have creds to login to docker hub as "civiform" user.
+#   Must have a GitHub API token with repo and read:org scopes located at "gh-release-token.txt"
+#
+# When successful, this script:
+#   1. tags the git commit for the release with the release number
+#   2. tags the docker image for the release with the release number
+#   3. creates a new draft release on github
+
+if not os.path.exists("gh-release-token.txt"):
+    print("gh-release-token.txt not found in root directory", file=sys.stderr)
```

Comment:
Not blocking: Consider using a multi-line string and combining the print statements:

```
print('''
gh-release-token.txt not found in root directory
Create one with repo....
''')
```

Reply:
done